mummer/4.0.0/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the following stage. We use `app` for the production image
+FROM ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG MUMMER_VER="4.0.0"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:xenial"
```

Comment:
if you upgrade the base image, please update this LABEL to reflect the new base image